src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -310,6 +320,18 @@ def container_options(entry: Entry, config: ApplicationConfiguration) -> PostPro
             entry.value.current = flatten_list(entry.value.current)
         return messages, exit_messages
 
+    @_post_processor
+    def exec_shell(self, entry: Entry, config: ApplicationConfiguration) -> PostProcessorReturn:
+        # pylint: disable=unused-argument
+        """Post process exec_shell.
```

Comment:
What is this `exec_shell` that you're post-processing here?
Also, non-code things will potentially be checked for spelling and since it's an identifier, it's best to at least mark it as such:
```suggestion
        """Post process ``exec_shell``.
```